README.md

```diff
@@ -1,40 +1,183 @@
+        1. Click "Auto Tests" if a plugin `*wkwebview*` is installed
+        1. Find out if the "permission buster" should be skipped (plugins splashscreen or inappbrowser, browser): `skipBuster`
+        1. Start polling in the background for events using the Web Driver (submitting `skipBuster` as well) every 2.5 seconds
+        1. Wait for the tests results
+            1. Time out if "connection takes to long" TODO (failure)  <!-- 8-479-->
+            1. Receive and handle "tests are done" (success) and "device disconnected" (failure) events<!-- 8-485-->
+        1. Log success or failure
+        1. Quit Web Driver
+        1. (platform = browser) Close the open browser <!-- 16-1056 -->
+        1. Close connection to Sauce Connect (Proxy)
+        1. Run the Appium tests on Sauce Labs (with sauce = true) <!-- 7-454 -->
+        </details>
+    - <details>
+      <summary>Run the Appium tests <!-- 6-379 --></summary>
+
+        1. Skip if action = build <!-- 6-384 -->
+        1. Skip is Appium should be skipped <!-- 6-388 -->
+        1. Skip if platform != android or ios <!-- 6-392 -->
+        1. !sauce: Error when no targetObj TODO <!-- 6-397 -->
+        1. Create Appium options <!-- 7-403 -->
+        1. Create AppiumRunner with options <!-- 7-426 -->
+            1. Prepare the submitted options <!-- AppiumRunner 151 -->
+            1. Create screenshot directory <!-- AppiumRunner 147 -->
+            1. Find the tests in plugin paths <!-- AppiumRunner 307 -->
+            1. Set globals for the tests <!-- AppiumRunner 334 -->
+        1. Skip if no Appium tests were found <!-- 7-427 -->  
+        1. Prepare App in AppiumRunner <!-- 7-433 -->
+            1. Remove server address from app
+            2. Reconfigure app (modify preferences + CSP, add plugin) <!-- 367, 375, 385 -- >
+            3. Build app
+        1. (sauce) Package and Upload the App to Sauce Labs <!-- 7-436 -->
+        1. Run tests via AppiumRunner <!-- 7-442 -->
+            1. (!sauce) Start iOS Proxy (`ios_webkit_debug_proxy`) <!-- AppiumRunner 231 -->
+            1. (!sauce) Install (`npm install appium`) <!-- AppiumRunner 231 --> and start Appium server <!-- AppiumRunner 252 -->
+            1. Start to run the Appium tests <!-- AppiumRunner 170 -->
+            1. Handle eventual exceptions, return the result
+      </details>
+1. <details>
+    <summary>Clean up</summary>
+
+    1. (!sauce) <!-- 2-107 -->
+        1. Handle timeouts of test execution above
+        1. Collect Device Logs
+        1. Uninstall App
+        1. Kill Emulator Process
+    1. (sauce) Display Sauce run details <!-- 2-118 -->
+    1. Clean up Project <!-- 2-121 -->
+    </details>
+
+## Installation
+
+```
+npm install -g cordova-paramedic
+
+# or
+
+git clone https://github.com/apache/cordova-paramedic
+# when cloning, you have to run `npm link` inside the checkout
+# or replace all occurences of `cordova-paramedic` in commands with `cordova-paramedic/main.js`
```

Comment:
My memory may be deceiving me, but isn't it `node cordova-paramedic/main.js`?

Reply:
Good point, it probably is on Windows in any case. 

Linux and Mac might be happy with 

https://github.com/apache/cordova-paramedic/blob/336ccca9f8d6290968f27a8e527084a102256aef/main.js#L1

Correct?